src/main/java/org/myrobotlab/framework/Platform.java

```diff
@@ -410,4 +406,9 @@ public static void main(String[] args) {
       // log.info("Exception: ", e);
     }
   }
+
+  public boolean getVmVersion() {
+    // TODO Auto-generated method stub
+    return false;

```

Comment:
What is this method for?  seems like it should return a string version.. or maybe an integer  number for a version..   remove this method if it's not used.

Reply:
yup ... I agree .. I started it, primarily because of the whole java 11 causing issues with missing dependencies and the split up of the jvm and the lawyers field day ... :P
But I apparently got distracted from the implementation - some of this info exists in the manifest anyway - I've updated my short list to take care of this ...